ansible/playbooks/roles/compatibility_layer/defaults/main.yml

```diff
@@ -16,7 +16,19 @@ gentoo_prefix_path: /cvmfs/{{ cvmfs_repository }}/versions/{{ eessi_version }}/c
 gentoo_git_repo: https://github.com/gentoo/gentoo.git
 # Select a specific commit in the gentoo_git_repo that should be used for the bootstrap,
 # e.g. by checking: https://github.com/gentoo/gentoo/commits/master
-gentoo_git_commit: 4ca74e7abe4f2b14e686267b517c59d43bb580b4
+# March 7 (4ca74e7abe4f2b14e686267b517c59d43bb580b4)
+# gentoo_git_commit: 4ca74e7abe4f2b14e686267b517c59d43bb580b4
+# April 4 (b9ebfbaef10f4740671c8bd90c6d2c929f8379bc) --> failed with circular dependencies (cmake, curl, nghttp2), cmake was update early on April 5
+# gentoo_git_commit: b9ebfbaef10f4740671c8bd90c6d2c929f8379bc
+# April 5 (8c0ce1bcd089d5badcb39afce98111bf4cfbc8f0)
+# gentoo_git_commit: 8c0ce1bcd089d5badcb39afce98111bf4cfbc8f0
+# March 13 (3a38f9b245e674722d5d58dfbb6baafca2b26975) commit which introduced
+# version 1.52.0 of nghttp2 (this commit switched to using cmake and likely
+# caused a circular dependency)
+# gentoo_git_commit: 3a38f9b245e674722d5d58dfbb6baafca2b26975
+# March 13 (7db97f5a3fc7ca8eced2a94e5f9481f7556cf908) one commit before
+# 3a38f9b245e674722d5d58dfbb6baafca2b26975
+gentoo_git_commit: 7db97f5a3fc7ca8eced2a94e5f9481f7556cf908
```

Comment:
Are all of these still relevant? Or can we clean them up and stick to a recent commit? I've used `29492845e41ea6a0a4a9769c7e0ce287d106079b` today and that one seems to work fine.

Reply:
Yes. we can clean it up. Only the last working one is needed.

Reply:
Cleaned up in https://github.com/EESSI/compatibility-layer/pull/177/commits/99014f0771bafff208da49798d65bab069a5a0bb